db/migrate/20170201020135_change_trips_table_to_allow_large_integers.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeTripsTableToAllowLargeIntegers < ActiveRecord::Migration[5.0]
```

Comment:
this allows it to load large integers successfully, although we may want to make a record that cuts all zipcodes down to 5 because i feel like this may be the issue.